packages/cli/test/unit/openapi/schema-model.unit.js

```diff
@@ -392,17 +410,27 @@ describe('schema to model', () => {
             signature: 'addresses?: AddressList;',
             decoration: "@property.array(Address, {name: 'addresses'})",
           },
+          {
+            name: 'us-office',
+            signature:
+              'usOffice?: {\n  street?: string;\n  city?: string;\n' +
```

Comment:
Wondering if the signature can be `usOffice: USAddress;` or `usOffice: Address` here instead.

Reply:
Good question. I'll check.

Reply:
It should be fixed. I have to recursively resolve `$ref` for some models.